Documentation/SpatialAwareness/SpatialObjectMeshObserver.md

```diff
@@ -1,79 +1,78 @@
+The spatial object mesh observer is an editor-only data provider for the Spatial Awareness system 
+that enables using an imported 3D model to represent the spatial mesh. One common use of the 
+spatial object mesh observer is to use data scanned via a Microsoft HoloLens to test how an
 experience adapts to different environments from within Unity.
 
 ## Getting started
 
-This guide will walk through setting up a *Spatial Object Mesh Observer*. There are three key steps to enable this feature.
+To enable the spatial object mesh observer, you will need to configure the spatial awareness system.
 
-1. Add a *Spatial Object Mesh Observer* to the Spatial Awareness system profile
-1. Set the Environment Mesh Data object
-1. [Configure rest of the Mesh Observer profile properties](ConfiguringSpatialAwarenessMeshObserver.md)
+1. Add a new spatial observer
 
-### Set up a *Spatial Object Mesh Observer* profile
+![Add Spatial Observer](../../Documentation/Images/SpatialAwareness/AddObserver.png)
 
-1. Select the desired *Mixed Reality Toolkit* configuration profile or select the *Mixed Reality Toolkit* object in scene
-1. Open or expand the *Spatial Awareness System* tab
-1. Click on *"Add Spatial Observer"* button
+2. Select the SpatialObjectMeshObserver 
 
-    ![Add Spatial Observer](../../Documentation/Images/SpatialAwareness/AddObserver.png)
+![Select Spatial Object Mesh Observer](../../Documentation/Images/SpatialAwareness/SelectObjectObserver.png)
 
-1. Select the *SpatialObjectMeshObserver* type
+3. Select the desired Spatial Mesh Object  
 
-    ![Select Spatial Object Mesh Observer](../../Documentation/Images/SpatialAwareness/SelectObjectObserver.png)
+![Select the Mesh Object](../../Documentation/Images/SpatialAwareness/ObjectObserverProfile.png)
 
-1. Select the desired *Spatial Mesh Object*. By default, the observer is configured with an example model. This model was created using a Microsoft HoloLens but it is possible to [create a new scan mesh object](#acquiring-environment-scans).
-1. [Configure rest of the Mesh Observer profile properties](ConfiguringSpatialAwarenessMeshObserver.md)
+By default, the observer is configured with an example model. This model was created using a Microsoft HoloLens.
 
-    ![Select the Mesh Object](../../Documentation/Images/SpatialAwareness/ObjectObserverProfile.png)
+## Acquiring environment scans
 
-### Spatial Object Mesh Observer profile notes
+### Windows Device Portal
 
-Since the *Spatial Object Mesh Observer* loads data from a 3D model, it does not honor some of the standard mesh
-observer settings which are outlined below.
+The [Windows Device Portal](https://docs.microsoft.com/en-us/windows/mixed-reality/using-the-windows-device-portal) can be used to download the spatial mesh, as a .obj file, from a Microsoft HoloLens device.
```

Comment:
```suggestion
The [Windows device portal](https://docs.microsoft.com/en-us/windows/mixed-reality/using-the-windows-device-portal) can be used to download the spatial mesh, as a .obj file, from a Microsoft HoloLens device.
```

Reply:
I think Windows Device Portal is considered branding maybe and is capitalized as-is in our platform docs:

![image](https://user-images.githubusercontent.com/3580640/69931962-1b9e1080-147e-11ea-85c0-62e27a9e3b27.png)

Reply:
i wasn't sure about that one - if official ms docs are putting it in uppercase we have to as well :) thanks for checking kurtis!